modules/nw-kuryr-cleanup.adoc

```diff
@@ -12,9 +12,15 @@ plugin, you must clean up the resources that Kuryr created previously.
 [NOTE]
 ====
 The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:
+
 * `openstacksdk` version 0.54.0
```

Comment:
```suggestion
* The `openstacksdk`Python package version 0.54.0
```

Reply:
Would it be acurate to say something like the above? I think this needs a "the" and I saw this elsewhere: https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.html#nw-kuryr-migration_migrate-from-kuryr-sdn